### PR TITLE
Update english-spanish-Part 2-Travel 2.csv

### PR DESCRIPTION
**Pull request by lazaridoue**
Updated words on Travel 2 part 2

### CODE REVIEW

**Comment by jmbeach** [comment]
Looks good, but the only thing I'd like changed is would you mind just deleting quiero from here? It's already in Part 1 Restaurants and Part 1 Shopping

File: english-spanish/Part 2/english-spanish-Part 2-Travel 2.csv [hunk 1]
```diff
@@ -1,14 +1,11 @@
 banana,banana
-quiero,I'll have
-quieres,you want
-quiere,she wants
```
Comment (jmbeach) [inline english-spanish/Part 2/english-spanish-Part 2-Travel 2.csv]: sorry, these two can stay, I just meant `quiero` itself

**Comment by lazaridoue** [comment]
Okay, I added those back

**Comment by jmbeach** [comment]
🎉 Thanks for contributing

**Pull request merged**